examples/browsertest/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# How to use this image
```

Comment:
This should be in examples/browser/test/mockrootdir
And we could add a Readme in there that says that these are just test files and are not actual code in theia...